core/src/apps/benfen/layout.py

```
@@ -0,0 +1,54 @@
+async def require_show_overview(
+    ctx: Context,
+    to_addr: str,
+    value: int,
+    currency_symbol: str = "BFC",
+) -> bool:
+    from trezor.strings import strip_amount
+
+    result = await should_show_details(
+        ctx,
+        title=_(i18n_keys.TITLE__SEND_MULTILINE).format(
+            strip_amount(format_benfen_amount(value, currency_symbol))[0]
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)